terraform/modules/monitoring/alert.tf

```diff
@@ -0,0 +1,132 @@
+resource "google_monitoring_notification_channel" "basic" {
+  project = var.project_id
+
+  display_name = "owner notification channel"
+
+  type = "email"
+
+  labels = {
+    email_address = var.notification_channel_email
+  }
+
+  force_delete = false
```

Comment:
I can't see `force_delete` in https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/monitoring_notification_channel

Reply:
Please reference here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_notification_channel#force_delete